mist-core/src/main/java/edu/snu/mist/core/rpc/DefaultMasterToTaskMessageImpl.java

```diff
@@ -38,17 +44,30 @@ private DefaultMasterToTaskMessageImpl(final GroupAllocationTable groupAllocatio
   }
 
   @Override
-  public Double getTaskLoad() {
-    // The list of event processors
-    // TODO: Synchronize eventProcessor update with master load update.
-    final List<EventProcessor> epList = groupAllocationTable.getKeys();
+  public TaskStats getTaskStats() {
```

Comment:
This method can be called concurrently with `GroupAllocationTableModifier` and there could be a possible concurrency issue.

Reply:
The inner `Map` and `Collect` in `GroupAllocationTable` supports concurrent access, so it would not throw exception on concurrent accesses.